lib/runner/variant-reassignment.js

```diff
@@ -175,6 +175,10 @@ export default class VariantReassignment {
         await this.transactionService.deleteTransaction(key)
       }
     } catch (e) {
+      this.logger.warn(
```

Comment:
Why is it warn here and below error?

Reply:
I put there `warn` because it is not the final error - as the reassignment will be retried a few times after which it will throw a final error.

But I am ok even with `error` level, what do you think?

Reply:
Let's leave it with `warn` for now.